modeldict/base.py

```diff
@@ -113,7 +113,7 @@ def local_cache_is_invalid(self):
             self.remote_cache_last_updated_key
         )
 
-        if not remote_last_updated:
+        if not isinstance(remote_last_updated, float):
```

Comment:
Shouldn't we preserve the initial check too? Ie.
```suggestion
        if not isinstance(remote_last_updated, float) or not remote_last_updated:
```

Reply:
```
In [43]: isinstance(None, float)                                                                                                   
Out[43]: False
```

Reply:
Heh, I meant more like:
```python
In [1]: remote_last_updated = 0.0

In [2]: bool(remote_last_updated)
Out[2]: False
```

Reply:
this variable is created by cache.get with no default so it'll either be a float (set from a valid timestamp) or None.